student/src/main/java/tds/student/performance/dao/TestOpportunityExamMapDao.java

```diff
@@ -0,0 +1,7 @@
+package tds.student.performance.dao;
+
+import java.util.UUID;
+
+public interface TestOpportunityExamMapDao {
```

Comment:
I know the performance work didn't do this but can you add JavaDocs to the interface so future people know what this is doing.  When I joined the project in August I would've really appreciated JavaDocs on the interfaces.  Instead I had to look at implementation to figure out what was happening, and there were times even that was hard to discern intent.